astropy/utils/introspection.py

```diff
@@ -111,6 +121,19 @@ def minversion(module, version, inclusive=True, version_path='__version__'):
     True
     """
 
+    if isinstance(module, types.ModuleType):
+        module_name = module.__name__
+    elif isinstance(module, six.string_types):
+        module_name = module
+        try:
+            module = resolve_name(module_name)
+        except ImportError:
+            return False
```

Comment:
right, I think we don't want to go overboard with the ``NUMPY_GT_1_6_BUT_LE_1_8_1_AND_ALSO_I_WANT_RASPBERRY_ICE_CREAM`` constants.  If it's not a common case just using `minversion` directly makes more sense.